src/components/input.js

```diff
@@ -225,11 +229,17 @@ const withAutocomplete = WrappedComponent => ({ value, onChange, suggestions: ra
     inputId: id
   }, [
     ({ getInputProps, getMenuProps, getItemProps, isOpen, openMenu, toggleMenu, highlightedIndex }) => {
-      return div({ style: { width: '100%', display: 'inline-flex' } }, [
+      return div({
+        onFocus: openOnFocus ? openMenu : undefined,
+        style: { width: '100%', display: 'inline-flex' }
```

Comment:
mob - Add shadow to the drop down, when overlapping other components it looks too flat and blends in an odd way. 
Check the workflow config, group management screen, and a few other places to make sure it is behaving correctly.

Reply:
Checked group config with the new code, l didn't see any issues. That is the only other spot in the code that I saw the autocomplete being used.